pydkron/client.py

```diff
@@ -128,7 +128,25 @@ def get_executions(self, name):
         """
         Get the job executions for a named job
         """
-        resp = self._call(_GET, "/executions/%s" % name)
+        resp = self._call(_GET, "/jobs/%s/executions/" % name)
         if resp.status_code == 404:
             raise DkronJobNotFound("Job %s was not found" % name)
         return resp.json()
+
+    def toggle(self, name):
+        '''toggle job
```

Comment:
Can `toggle job` from the comment, the `Enable / disable job` comment below is enough

Reply:
Got it.